docs/source/MineYourMind/index.rst

```diff
@@ -20,19 +22,31 @@ Performance and Stability
-If you are playing any of our other modpacks, We have Golden Shovel Protection. You start off with 100 Claim blocks. You can gain more from voting or donating. So the longer you play the more claim blocks you get so you can expand your claim. If you are wanting to start a claim then you should have a Golden Shovel when you start. If you Equip it, It will tell you how many claim blocks you have in chat. Then you right click the ground where you want to start your claim. Then in the Opposite Corner right click again. This will create your claimed area (Providing you have enough Claim blocks) You can use the command /trust (playername) To allow them to edit inside your claim. You can also use /abandonclaim (While standing in your claim) to delete that claim or /abandonallclaims to remove every claim you own. You can view a tutorial on how to use this here (www.youtube.com/watch?v=LAEp7KTgIWo)
+Our servers run two different types of Grief Protection. If you are playing on a Skyblock/Custom Map Modpack your islands/Area are automatically claimed for you. To change the features of this claim you can use the command ``/is``. Using this command allows you to set your island home, View your team members, Change the Biome and it comes with other features as well.
+
+There is no changing how much is claimed so you need to make use of your space. Remember the claim goes Down to bedrock and up to the build limit.
+
+
+If you are playing any of our other modpacks, We have Golden Shovel Protection. You start off with 100 Claim blocks, and you can gain more from voting or donating. So the longer you play the more claim blocks you can get to expand your claim. 
+
+If you want to make a claim, then you can use the Golden Shovel that you started the pack with, or craft a new one if you lost it. When you Equip it, It will tell you how many claim blocks you have in the chat. Then you right click the ground where you want to start your claim. Then in the Opposite Corner right click again.
+
+This will create your claimed area (Providing you have enough Claim blocks) You can use the command ``/trust <playername>`` To allow them to edit inside your claim. You can also use ``/abandonclaim`` (While standing in your claim) to delete that claim or ``/abandonallclaims`` to remove every claim you own. 
+
+You can view a tutorial on how to use this here: www.youtube.com/watch?v=LAEp7KTgIWo
 
 Staff
 =====
 
 We currently have 6 Ranks in our staff Team. It follows as Jr.Mod -> Mod -> Sr.Mod -> Manager -> Admin -> Founder
-We also have a couple irregular Ranks like Consultant & Developer
+We also have a couple irregular Ranks like Consultant, Developer and Community Coordinator.
```

Comment:
Were you going to add Community Coordinator description out of curiosity?

Reply:
Kinda forgot about this update for a while.

I comitted an update to my branch, which i think was how i'm supposed to make changes to this pull request.

I added the fixes that you suggested, and also added the Community Coordinator to the ranks.